src/app/events/services/evaluation-default-grade-update.service.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { provideHttpClient, withFetch } from "@angular/common/http";
+import { TestBed } from "@angular/core/testing";
+import { of } from "rxjs";
+import { buildGradingItem } from "src/spec-builders";
+import { buildTestModuleMetadata } from "src/spec-helpers";
+import { GradingItemsRestService } from "../../shared/services/grading-items-rest.service";
+import { EvaluationDefaultGradeUpdateService } from "./evaluation-default-grade-update.service";
+import { EvaluationStateService } from "./evaluation-state.service";
+
+describe("EvaluationDefaultGradeUpdateService", () => {
+  let updateService: EvaluationDefaultGradeUpdateService;
+  let gradingItemsRestServiceMock: jasmine.SpyObj<GradingItemsRestService>;
+  let evaluationStateServiceMock: jasmine.SpyObj<EvaluationStateService>;
+  const mockEvent = {
+    id: 1000,
+    designation: "Clowns 101",
+    type: "course",
+    studentCount: 24,
+    gradingScaleId: 10000,
+  };
+
+  const mockGradingItem1 = buildGradingItem(10001, 100001);
+  const mockGradingItem2 = buildGradingItem(10002, 100002);
+  const mockGradingItems = [mockGradingItem1, mockGradingItem2];
+  mockGradingItems.map((item) => ({
+    ...item,
+    IdGrade: 200001,
```

Comment:
Nicht, dass es jetzt hier gross Auswirkungen hätte, aber du nimmst hier die gleiche IdGrade, welche du unten in Update-Test setzt. Für ein etwas realistischeres Beispiel, würde ich schauen, dass du hier für die Einträge eine andere IdGrade wählst.